File: webby/content/ruby/hello-world.rb
require 'java'

$CLASSPATH << 'C:\\blp\\API\\APIv3\\JavaAPI\\v3.3.1.0\\lib\\blpapi3.jar'
$CLASSPATH << "blpwrapper.jar"

include_class "org.findata.blpwrapper.Connection"

conn = Connection.new

securities = ["GOOG US Equity", "OCN US Equity"].to_java(:string)
fields = ["NAME", "PX_LAST", "TIME"].to_java(:string)

result = conn.blp(securities, fields)
puts result.getData.to_a.inspect

File: webby/content/python/hello-world.py
import sys

sys.path.append('C:\\blp\API\APIv3\JavaAPI\lib\\blpapi3.jar')
sys.path.append("blpwrapper.jar")

from org.findata.blpwrapper import Connection

conn = Connection()
result = conn.blp(["RYA ID Equity"], ["NAME"])

print(result.getData()[0][0])

